E1a-ARSE/ALSE/README.md

```diff
@@ -0,0 +1,171 @@
+
+
+## Summary (Intent)
+
+This is the test instruction for IFC4x3_AbRV-E1a-ALSE - Alignment Superelevation & Width.
+
+The test instruction includes:
+
+- A basic project structure setup including units and global positioning
+- An alignment structure for a road including the horizontal and vertical layouts
+- Assignment of superelevation- and width events along the alignment geometry using linear placement marking the locations where these design parameters change and the values of these design parameters
+
+The data comes from, and is a simplified version of, the IFC Infra Unit Test [MCON-2](https://github.com/bSI-InfraRoom/IFC-infra-unit-test/tree/main/MCON-2) which in turn is extracted from the IFC Rail [Level Crossing storyline](https://github.com/IFCRail/IFC-Rail-Unit-Test/tree/master/8_Storylines%20Test%20(SL)/SL08_Level%20Crossing). 
+
+...
+
+The [Expected Results](#Expected-Results) section lists the material that will be used to assess the fulfilment of capabilities.
+
+:zap: **This is a test-driven process: refer to the [Validation Criteria](#Validation-Criteria) to understand what is required by the test** :zap:
+
+## Itemised Roots
+*This section lists the primary entities and concept templates being validated by this test instruction, these are listed to understand the scope of the data entities being addressed. this is the documentation of the data placed within BIMQ*
+
+The Test instruction addresses the import and export of the following IFC Entities & Concept Templates:
+
+:construction: under construction :construction:
+
+<details><summary>IFC Entities</summary>
+
+These entities represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD. **The scope of the test shall not be used as a definitive scope of the exchange, or of the entire MVD.**
+
+- Inherited from impported tests:
+  - *IfcProject*
+  - *IfcSite*
+  - *IfcRoad*
+  - *IfcAlignment*
+  - *IfcAlignmentHorizontal*
+  - *IfcAlignmentVertical*
+  - *IfcAlignmentSegment*
+  - *IfcAlignmentHorizontalSegment*
+  - *IfcAlignmentVerticalSegment*
+  - *IfcContext*
+
+- For this test instruction:
+  - *IfcFacilityPart*
+  - *IfcAnnotation*
+  - *IfcPropertySet*
+  - *IfcPropertySingleValue*
+  - *IfcPropertyEnumeratedValue*
+
+
+</details>
+
+<details><summary>Concept Templates</summary>
+
+These concept templates represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD, that must be correctly exported to meet the validation criteria. **The scope of the test shall not be used as a definitive scope of the exchange, or of the entire MVD.**
+
+- Inherited from imported tests:
+  - *Project Units*
+  - *Project Representation Context*
+  - *Project Global Positioning*
+  - *Spatial Decomposition*
+  - *Spatial Composition*
+  - *Spatial Container*
+  - *Alignment Decomposition*
+  - *Alignment Geometry Gradient*
+  - *Product Local Placement*
+
+- For this test instruction:
```

Comment:
Would expect this one would be inheriting from position stationing. In that case, these two templates would be inherited.

Reply:
For now, I went by the TI dependencies specified but let's discuss.